sql-schema/127.sql

```diff
@@ -0,0 +1,11 @@
+DROP TABLE IF EXISTS `ipmi_sensors`;
```

Comment:
as per what is does:

this is the default ipmi output (to me it looks pretty useless because of their names):
[
<img width="1044" alt="screen shot 2016-08-04 at 12 03 11 am" src="https://cloud.githubusercontent.com/assets/3048315/17381930/1bfb4472-59d7-11e6-9a3a-281abac59470.png">
](url)

This is the "new" look:
<img width="941" alt="screen shot 2016-08-04 at 12 11 54 am" src="https://cloud.githubusercontent.com/assets/3048315/17382147/1c058512-59d8-11e6-85a4-01842c4e1c1b.png">